action-src/register.js

```diff
@@ -1,6 +1 @@
-const Observable = require('zen-observable');
-
-global.Observable = Observable;
-require('any-observable/register')('global.Observable');
-
```

Comment:
This warrants a major version bump I think.

Reply:
I'm still registering an observable in the node code

Reply:
Here: https://github.com/chromaui/chromatic-cli/blob/39642116974da41d4c023ec547bbee4aab8689ab/node-src/index.ts#L3